README.md

```diff
@@ -100,12 +100,12 @@ jobs:
         uses: neofinancial/ticket-check-action@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          ticketLink: 'https://app.clubhouse.io/:org/story/%ticketNumber%'
-          ticketPrefix: 'CH-'
-          titleRegex: '^(CH)(-?)(?<ticketNumber>\d{3,})'
-          branchRegex: '^(CH)(-?)(?<ticketNumber>\d{3,})'
-          bodyRegex: '(CH)(-?)(?<ticketNumber>\d{3,})'
-          bodyURLRegex: 'http(s?):\/\/(app.clubhouse.io)(\/:org)(\/story)\/(?<ticketNumber>\d+)'
+          ticketLink: 'https://app.shortcut.com/:org/story/%ticketNumber%'
+          ticketPrefix: 'sc-'
+          titleRegex: '^(CH|sc)(-?)(\d+)\/.+'
+          bodyRegex: '(CH|sc)(-?)(?<ticketNumber>\d+)\/.+'
+          branchRegex: '^(CH|sc)(-?)(?<ticketNumber>\d+)\/.+'
+          bodyURLRegex: 'https?:\/\/(app\.(clubhouse|shortcut)\.(io|com))\/neofinancial\/story\/(?<ticketNumber>\d+)\/.+'
```

Comment:
You can probably combine the `(clubhouse|shortcut)` and `(io|com)` groups. You should also put back the `(\/:org)` group.

Reply:
Was thinking a bit more on this one and I am not sure if it's worth to even keep the clubhouse.io regex. We already can check for older branches with body/branch regexes and since they already updated their domain, think it'll be almost impossible to have a clubhouse.io link somehwre.

Reply:
About the `(\/:org)` I am not sure this is working as expected. Made two test scenarios that I think proves my point. It is not being by the code as well, the only matching group we use is `ticketNumber`

<img width="754" alt="Screen Shot 2021-09-15 at 12 47 57" src="https://user-images.githubusercontent.com/86978189/133466444-a3c48af3-5f44-41d3-af20-01d9edc14f65.png">

<img width="883" alt="Screen Shot 2021-09-15 at 12 48 11" src="https://user-images.githubusercontent.com/86978189/133466496-bb5b7e73-b422-4ba3-9575-1cf8b9e5e75a.png">

Reply:
Ahh yeah the :org is just a placeholder for the README, not a regex thing. It's so this is more friendly to folks outside Neo.